packages/bull-vault/src/BullStrategy.sol

```diff
@@ -110,8 +111,10 @@ contract BullStrategy is ERC20, LeverageBull {
      * @param _receiver receiver address
      */
     function farm(address _asset, address _receiver) external onlyOwner {
+        require(!IController(powerTokenController).isShutDown(), "BS7");
+        require(!hasRedeemedInShutdown, "BS12");
```

Comment:
same for this one

Reply:
!hasRedeemedInShutDown => !isShutDown (contrapositive) so we only need one of these. I guess per the comment above this is to stop the owner farming out all the eth in shutdown and rugging users (is that right?). Makes sense if so, but only need to stop that specifically.